presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriverUri.java

```diff
@@ -84,6 +84,7 @@
 public final class PrestoDriverUri
 {
     private static final String JDBC_URL_START = "jdbc:";
+    private static final String JDBC_SCHEME = "presto";
```

Comment:
We have this validation in PrestoDriver, see `io.prestosql.jdbc.PrestoDriver#DRIVER_URL_START`

Of course, now that `PrestoDriverUri` is public, it needs its own validation.

- make the `Make PrestoDriverUri accessible outside of package` commit last, since all others are actually prerequisites for that
- in this validation here, use `io.prestosql.jdbc.PrestoDriver#DRIVER_URL_START` const (move it from driver to here)